### DISCUSSION TITLE
Corrupted Yml file

### DISCUSSION THREAD

**Discussion by jacksoderberg** [EdgeTX/edgetx#2601] [Q&A]
Upon editing a model in the Tx16s suddenly all widgets was gone on that specific model, radio was restarted but still a screen without anything expect for the backround, trying load models in companion suddenly this message pops up, anyone that can help me out?

Cheers
Jack
<img width="1280" alt="Skärmavbild 2022-10-18 kl  21 54 12" src="https://user-images.githubusercontent.com/115811044/196531544-ac7978c3-2ceb-4f58-9245-53f11a00ea6e.png">

**Comment by pfeerick** [comment]
Can you zip and upload your RADIO and MODELS folders from the radio's SD card to here (just drag them to the text editor area once you have the zip file). 

Do you have any idea what you were editing on the radio before it went silly? And what firmware version are you running?

**Reply by jacksoderberg** [reply]
Hello
Im using
Edge 2.7.1
Companion 2.7.1

Im sorry but cant't upload because after posting it kind of started to work normally but just almost.
I was messing around with different widgets and settings when suddenly the radio went in emergency mode, after restarting I continued with the widgets but skipped my own taken model picture, before this upon setting widgets  I restarted mine radio again and all widgets where gone, just a blank background screen, I did this 3 times and always the widgets where gone, on the 4 attempt i skipped mine own taken model picture and the everything seemed normal until i was to continue with mine fysical model that is a ship with 14 channel setup, the binding where lost, channel 1-4 , ail,elev and rudder in inputs where gone, the throttle was on the right stick as in mode 1 (Im in mode 2)
After rebinding and manually putting back the channels on inputs for now seems to be normal and also loading in models in companion works, really dont what was messing upp things but, maybe mine model picture or something else, just dont know for now
Cheers
Jack
